src/content/fr/fundamentals/performance/rendering/index.md

```diff
@@ -0,0 +1,155 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Les utilisateurs remarquent quand les sites ou applications ne
+fonctionnent pas correctement, leur optimisation est donc cruciale!
+
+{# wf_updated_on: 2018-08-17 #}
+{# wf_published_on: 2015-03-20 #}
+{# wf_blink_components: Blink>Paint #}
+
+# Performance de rendu {: .page-title}
+
+{% include "web/_shared/contributors/paullewis.html" %}
+
+Les utilisateurs actuels d'Internet
+[s'attendent à ce que les pages qu'ils visitent soient
+interactives](https://paul.kinlan.me/what-news-readers-want/)
+et c'est là que vous devez concentrer votre temps et vos efforts. Les pages ne
+doivent pas seulement se charger rapidement, elles doivent aussi fonctionner
+correctement; le défilement doit être très rapide, les animations et
+interactions doivent être aussi intuitives que possible.
+
+De manière à écrire des sites et des applications performantes il vous faut
+comprendre comment le HTML, le Javascript et le CSS sont gérés par le
+navigateur, et vous assurer que le code écrit (et le code tiers que vous
+incluez) fonctionne le plus efficacement possible.
+
+## 60 ips et taux de rafraîchissement des appareils
```

Comment:
I think we can use FPS as it is common term (even in french).  WDYT?

Reply:
Agree, FPS is an equivalent of IPS for me in French and can be easily understood.